.gitignore

```diff
@@ -3,6 +3,7 @@
 /public/storage
 /public/js
 /public/css
+/public/images
```

Comment:
not sure if this is to be ignored, does webpack compile images here? @m-triassi

Reply:
@willyyhuang It does, yeah, images we want to keep in the repo (say for icons and such) should be live in `resources`